charts/csm-application-mobility/Chart.yaml

```diff
@@ -6,13 +6,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.0
+version: 0.4.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.3.0"
+appVersion: "0.4.0"
```

Comment:
@xuluna any reason for bumping this version? In past releases, we haven't released a new version for app mobility

Reply:
Because I updated the image of image: dellemc/csm-application-mobility-controller from 0.3.0 to 0.4.0. It has been using an old version of image. Should the app version be still the old one?

Reply:
I'm not sure if there will be a new image version for app mobility and encryption for 1.11, have to check with CPO

Reply:
These should be old existing versions instead of new ones. Upon checking dockerhub again I found these tags have been deleted. I can revert back to original versions. Thanks for catching these.